### PR TITLE
Create outline for Boston file

### PR DESCRIPTION
**Pull request by abbasms**
@ravisha1990hpe @mayankbothra15
Check this out.

### CODE REVIEW

File: abbasms/abbasms-githubteacher-boston.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Boston
+
+### Restaurants
+
```
Comment (dkalashian) [inline abbasms/abbasms-githubteacher-boston.md:7]: LaFamiligia in the north end

**Comment by abbasms** [comment]
Random comment

File: abbasms/abbasms-githubteacher-boston.md (hunk 1 above)
Comment (jkamal) [inline abbasms/abbasms-githubteacher-boston.md:7]: How about things to do...@abbasms :walking:

Comment (pritish-devurkar) [inline abbasms/abbasms-githubteacher-boston.md:7]: Boston Common :walking: ! @jkamal @abbasms @dkalashian

**Comment by abbasms** [comment]
Closes issue #41

**Pull request merged**